plaidml/edsl/__init__.py

```diff
@@ -672,6 +682,16 @@ def inputs(self):
     def outputs(self):
         return [x for x in self.args if not x.is_input]
 
+    @property
+    def passes(self):
+        """Returns a list of passes in to form: [(name, ir), ...]
```

Comment:
Thanks for writing up a docstring about this API. We'll probably want to regenerate documentation after this lands. That reminds me that we wanted to start auto-rebuilding ReadTheDocs at some point. I've created a task in our planner for that.

Reply:
I'll try to make another pass at the docs at some point before the demo and C4ML so we have something good to generate.